terraform/eks/container-insights-agent/logs_sample_fargate.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flblogproducer2eks
+  labels:
+    app: flblogproducer2eks
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: flblogproducer2eks
+  template:
+    metadata:
+      labels:
+        app: flblogproducer2eks
+    spec:
+      containers:
+        - name: flblogproducer2eks
+          #image: 803860917211.dkr.ecr.us-west-2.amazonaws.com/alpine-log-producer:latest
+          image: 803860917211.dkr.ecr.us-west-2.amazonaws.com/python-logger:300sec
```

Comment:
This image is in a personal ECR account. Let's replace it with `nginx:latest` and remove the command section.